src/index.js

```diff
@@ -356,7 +356,7 @@ export default class OneSignal {
     static removeExternalUserId(handler) {
         if (!isObjectNonNull(RNOneSignal)) return;
 
-        if (handler === undefined)
+        if (handler === undefined && Platform.OS === 'ios')
```

Comment:
Can you add a comment above this change so we can keep a note about this:

> // Android workaround for the current issue of callback fired more than once

Reply:
Sure, will do

Reply:
@nan-li Added the comment.